sdk/include/psdk/windef.h

```diff
@@ -178,7 +178,7 @@ typedef int *PINT;
 typedef int *LPINT;
 typedef WORD *PWORD;
 typedef WORD *LPWORD;
-#ifndef __ROS_LONG64__
+#if !defined(__ROS_LONG64__) || defined(_M_AMD64)
```

Comment:
Nice! I like how _M_AMD64 doesn't need to buy into this crappy hack. I guess I did something right earlier.

Reply:
I am just following the other ifdef that already does this.
There is a third, but it does do something completely different again...

Reply:
I'd say remove that other ifdef. There should be no reason why this would work differently between 32 and 64 bit.

Reply:
done.